libminifi/include/FlowController.h

```diff
@@ -90,6 +93,25 @@ class FlowController
 		}
 		return _flowController;
 	}
+	//! passphase for the private file callback
```

Comment:
All the central service like provenance and others are tied with flowController like current NiFi.
It make code more clear.